src/data-fetcher-loop/data-fetcher-loop.ts

```diff
@@ -77,32 +75,33 @@ export const runDataFetcher = async () => {
     const signedDataFetchIntervalMs = signedDataFetchInterval * 1000;
 
     // Better to log the non-decomposed object to see which URL comes from which chain-provider group.
-    logger.debug('Fetching data from signed APIs.', { signedApiUrls });
+    logger.debug('Signed API URLs.', { signedApiUrls });
     const urls = uniq(
       Object.values(signedApiUrls)
         .map((urlsPerProvider) => Object.values(urlsPerProvider))
         .flat(2)
     );
 
+    const urlCount = urls.length;
+    const staggerTimeMs = signedDataFetchIntervalMs / urlCount;
+    logger.info('Fetching signed data', { urlCount, staggerTimeMs });
```

Comment:
I'm really curious how many Signed API calls do we make, so I've replaced the nearly useless `Running data fetcher.` message with this.